.gitignore

```diff
@@ -1,2 +1,4 @@
 test.md
 /node_modules
+.hintrc
+stylelintrc.json
```

Comment:
- [ ] [Optional] It's not a good to idea to put these files on gitignore because they help a lot when you want to run linters check on your local machine